packages/core/src/App/Containers/ConfirmEmailModal/confirm-email-modal.tsx

```diff
@@ -60,7 +71,6 @@ export const ConfirmEmailModal = observer(
                 title={<Localize i18n_default_text='Are you sure?' />}
                 toggleModal={onClose}
                 width='440px'
-                is_closed_on_cancel={false}
                 has_close_icon={false}
```

Comment:
can we remove this as well since we set `has_close_icon={false}`

Reply:
@suisin-deriv , I didn.t get it, why should we remove it?
I removed is_closed_on_cancel because there is no sich option in Modal, just wrong props passed

Reply:
Owhhh okok understand, you can ignore this comment 🙏